ibis/pandas/core.py

```diff
@@ -302,6 +364,8 @@ def main_execute(expr, params=None, scope=None, aggcontext=None, **kwargs):
         The data that an unbound parameter in `expr` maps to
     scope : Mapping[ibis.expr.operations.Node, object]
         Additional scope, mapping ibis operations to data
+    timecontext : TimeContext
```

Comment:
Optional

Reply:
Just realized that in definition TimeContext is already Optional. I have made all timecontext typings match with each other.